Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile
+FROM node:10-alpine
```

Comment:
should be harmonised with .nvmrc: https://github.com/algolia/algoliasearch-client-javascript/blob/master/.nvmrc